.github/workflows/main.yaml

```diff
@@ -30,14 +40,13 @@ jobs:
         uses: actions/setup-dotnet@v1
         with:
           dotnet-version: 6.0.x
+      # Adjustment is done prior to Nuke build as OCTOVERSION information is included in the result package.
+      - name: Append OCTOVERSION_CurrentBranch with -nightly (for scheduled)
+        if: github.event_name == 'schedule'
+        run: echo "OCTOVERSION_CurrentBranch=${{ env.OCTOVERSION_CurrentBranch }}-nightly" >> $env:GITHUB_ENV
       - name: Nuke Build 🏗
         id: build
         shell: bash
-        # Pass branch and patch number to Nuke OctoVersion
-        # (for pull_request events we override the /refs/pull/xx/merge branch to the PR's head branch)
-        env:
-          OCTOVERSION_CurrentBranch: ${{ github.head_ref || github.ref }}
-          OCTOVERSION_Patch: ${{ github.run_number }}
```

Comment:
Did we mean to remove this line? We don't want to be generating the same package version more than once, even if we re-run a build.